service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToSubmitTest.java

```diff
@@ -121,4 +125,33 @@ void shouldRetainPaymentInformationInCase() {
             .containsEntry("amountToPay", "233300")
             .containsEntry("displayAmountToPay", YES.getValue());
     }
+
+    @Nested
+    class LocalAuthorityValidation {
+
+        @Test
+        void shouldReturnErrorWhenCaseBelongsToSmokeTestLocalAuthority() {
```

Comment:
not perfect name, it is not about smoke test LA, it is about any org that has toggle set to false. From this test you cannot have a clue that this is controlled by toggle